pytest_reportportal/service.py

```diff
@@ -92,10 +91,10 @@ def init_service(self, endpoint, project, uuid, log_batch_size,
                 project=project,
                 token=uuid,
                 error_handler=self.async_error_handler,
-                log_batch_size=log_batch_size,
-                verify_ssl=verify_ssl
+                log_batch_size=log_batch_size  # ,
+                # verify_ssl=verify_ssl
```

Comment:
Please, un-comment these lines or explain the reason to merge this PR as-is.

Reply:
I had to implement the fix of PR #112 in order to test my changes in a live env - otherwise, I would get an exception right away that `ReportPortalServiceAsync` class does not have any argument named `verify_ssl`